sdk/src/main/java/io/dapr/client/ObjectSerializer.java

```diff
@@ -22,17 +22,22 @@
  */
 public class ObjectSerializer {
 
+  private final ObjectMapper objectMapper;
```

Comment:
To create my custom serializer I saw on DefaultObjectSerializer. In my case, I duplicate ObjectSerializer class and change only ObjectMapper. I thought that makes sense to add more flexibility to avoid code duplication. I do not understand why we should protect ObjectSerializer class

Reply:
Like in the comment below, copying the DefaultObjectSerializer is not really a concern. The concern for us is the support of an increased API surface by signing up to support a specific serialization solution. I am OK if the community wants to have an optional "dapr-serialization"  or "dapr-jackson" packages with common serialization solutions that the community wants to support, but it should not be part of the core SDK.